ledger/artifactmanager/jettreeupdater.go

```diff
@@ -109,7 +110,10 @@ func (jtu *jetTreeUpdater) fetchJet(
 		jtu.seqMutex.Unlock()
 	}()
 
-	resJet, err := jtu.fetchActualJetFromOtherNodes(ctx, target, pulse)
+	ch := jtu.fetchActualJetFromOtherNodes(ctx, target, pulse)
```

Comment:
it's possible to avoid returning channel there

Reply:
done